src/app/ipv/middleware.js

```diff
@@ -285,6 +285,7 @@ module.exports = {
         case "pyi-timeout-recoverable":
         case "pyi-timeout-unrecoverable":
         case "pyi-f2f-technical":
+        case "pyi-kbv-fail":
```

Comment:
Hesitant to name it `fail` when the dropout route isn't a failure as such - maybe `pyi-kbv-exit` ?

Reply:
@RogVanGDS I'm wondering if we should even reference no photo id given it's a no photo id specific page - maybe something like `pyi-kbv-escape-no-photo-id` despite being a mouthful. The number of pages around CRI escape is already getting confusing so good to be as descriptive as possible. Thoughts?